ui/src/organizations/components/CreateOrgOverlay.tsx

```diff
@@ -1,5 +1,6 @@
 // Libraries
 import React, {PureComponent, ChangeEvent} from 'react'
+import _ from 'lodash'
```

Comment:
I've been seeing more of the `import {sample} from 'lodash'` style and the default import. I don't know which we prefer.